### PR TITLE
Add new guest speakers

### LINKED ISSUE #18: Add new guest speakers
- [ ] Create slider/caroussel for gueste speakers
- [ ] Add new guest speakers: Gabriela Davila and Evandro Mohr

### PR DESCRIPTION
**Pull request by alessandrofeitoza**
This resolve #18 

Add guest speakers: Evandro Mohr and Gabriela Davila.

### CODE REVIEW

File: 2019/assets/sass/paper-kit/_carousel.scss [hunk 1]
```diff
@@ -82,5 +82,8 @@
             padding-left: 3px;
         }
         }
```
Comment (cgrandval) [inline 2019/assets/sass/paper-kit/_carousel.scss]: Remove unnecessary bracket

File: 2019/index.html [hunk 2]
```diff
@@ -199,51 +199,170 @@ <h5 class="description">Conheça nossos ilustres palestrantes convidados, escolh
...
-                        <div class="card-footer text-center">
-                            <a href="https://www.linkedin.com/in/tdutrion/" target="_blank" class="btn btn-just-icon btn-linkedin"><i class="fa fa-linkedin"></i></a>
-                            <a href="https://www.twitter.com/tdutrion" target="_blank" class="btn btn-just-icon btn-twitter"><i class="fa fa-twitter"></i></a>
-                            <a href="https://www.github.com/tdutrion" target="_blank" class="btn btn-just-icon btn-github"><i class="fa fa-github"></i></a>
-                        </div>
-                    </div>
-                </div>
+                <div class="col-md-12 ml-auto mr-auto">
+                    <div class="card page-carousel">
+                        <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
```
Comment (cgrandval) [inline 2019/index.html]: Why Example in the name :-)

File: 2019/index.html [hunk 3]
```diff
@@ -199,51 +199,170 @@ <h5 class="description">Conheça nossos ilustres palestrantes convidados, escolh
...
+                            <div class="carousel-inner" role="listbox">
+                                <div class="carousel-item active">
+                                    <div class="row">
+                                        <div class="col-1"></div>
+                                        <div class="col-5">
+                                            <div class="card card-profile" data-background="color" data-color="dark">
+                                                <div class="row">
+                                                    <div class="col-md-8">
+                                                        <div class="card-img-top">
+                                                            <a href="#pablo">
```
Comment (cgrandval) [inline 2019/index.html]: Who is pablo lol

**Review by cgrandval — CHANGES_REQUESTED** [review]

File: 2019/assets/sass/paper-kit/_carousel.scss (hunk 1 above)
Comment (alessandrofeitoza) [reply 2019/assets/sass/paper-kit/_carousel.scss]: but is necessary, this bracket close the "a" block.

**Review by cgrandval — APPROVED** [review]

**Pull request merged**